.github/workflows/runtime-sync.yml

```diff
@@ -0,0 +1,84 @@
+name: AspNetCore-Runtime Code Sync
+on:
+  # Test this script using on: push
+  # push
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#scheduled-events-schedule
+    # Once per day at midnight PST (8 UTC)
+    - cron: '0 8 * * *'
+
+jobs:
+  compare_repos:
+    name: Compare the shared code in the AspNetCore and Runtime repos and notify if they're out of sync.
+    runs-on: windows-latest
+    steps:
+    - name: Checkout aspnetcore
+      uses: actions/checkout@v2.0.0
+      with:
+        # Test this script using changes in a fork 
+        # repository: 'Tratcher/aspnetcore'
+        repository: 'dotnet/aspnetcore'
+        path: aspnetcore
+    - name: Checkout runtime
+      uses: actions/checkout@v2.0.0
+      with:
+        # Test this script using changes in a fork 
+        # repository: 'Tratcher/runtime'
+        repository: 'dotnet/runtime'
+        path: runtime
+    - name: Copy
+      shell: cmd
+      working-directory: .\runtime\src\libraries\Common\src\System\Net\Http\aspnetcore\
+      env:
+        ASPNETCORE_REPO: d:\a\aspnetcore\aspnetcore\aspnetcore\
```

Comment:
Is `d:\a\` guaranteed to stay the root path? Can we use a relative path instead? Or pull the root path out of the environment like you do for secrets.GITHUB_TOKEN?

Also, is it really really cloned to `aspnetcore\aspnetcore\aspnetcore\`? 🤣

Reply:
These are based on experimental results, the docs don't cover details like this. I'll leave it long enough to find out if it holds true in another repo.

> Also, is it really really cloned to aspnetcore\aspnetcore\aspnetcore\? 🤣

Yeah... the last `aspnetcore\` is because I'm cloning aspnetcore and runtime into subfolders side-by-side. Normally they'd expect you to checkout your own repo into the root directory.  Not sure why there's a second aspnetcore before that though.

Reply:
I tested this some more and we're only allowed to edit files in `d:\a\aspnetcore\aspnetcore\`. That means we have to clone runtime and aspnetcore as subfolders.

The root path has been stable so I'm not going to spend more effort on it for now.